test/elements.test.ts

```diff
@@ -0,0 +1,44 @@
+import { ElementDictionary } from "../src/data/elements";
+import { Phases, Phase } from "../src/interfaces/element-props";
+
+function checkUrl(url: string): boolean {
+  let isUrl = true;
+  try {
+    new URL(url);
+  } catch (e) {
+    isUrl = false;
+  }
+  return isUrl;
+}
+
+function checkPhase(phase: Phase): boolean {
+  return Object.values(Phases).includes(phase);
+}
```

Comment:
Why is this  global function needed when we are only using in in one place? we can directly do  
```js
expect(Object.values(Phases).includes(ElementDictionary[el].phase));).toBe(true)
```

Reply:
Yes you are right about it sumit , but doesn't it make the code modular and more readable. @maneetgoyal sir should we keep the function or go with sumit's suggestions.

Reply:
To make it simpler destructuring can also be used.
```js
const {phase} = ElementDictionary[el]
expect(Object.values(Phases).includes(phase)).toBe(true);
```

Reply:
>should we keep the function or go with sumit's suggestions.

Function is fine